boilerPlate/src/app/modules/private/private-layout/private-layout.component.html

```diff
@@ -0,0 +1,4 @@
+<!-- <app-sidebar></app-sidebar>  Uncomment when require sidebar -->
+<app-header></app-header>
+<router-outlet></router-outlet>
```

Comment:
Can also add a side-menu with header and footer.

Reply:
User can set whatever is the requirement